portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,57 @@ function myFunction() {
     x.className = "nav";
   }
 }
+
+function getHello() {
+  console.log('Fetching a random message.');
```

Comment:
You don't use this code anymore right? If so remove.

Reply:
I do not need these functions anymore. I also went ahead and delete that section on index.html.